lib/takeaway.rb

```diff
@@ -0,0 +1,70 @@
+require 'twilio-ruby'
+
+# put your own credentials here
+account_sid = 'AC71ede767f3f69bf43718d8094b244800'
+auth_token = '88ac7f43909c00b5b8e9a32d5b554c35'
+
+# set up a client to talk to the Twilio REST API
+@client = Twilio::REST::Client.new account_sid, auth_token
+
+class Menu
+  def initialize
+    @list_of_dishes = []
+  end
+  def list_of_dishes
+    @list_of_dishes
+  end
```

Comment:
This is fine; you could also use `attr_reader :list_of_dishes` to achieve the same effect.

Reply:
Also, there's a way to build the program that doesn't involve `Menu` spilling its secrets (= allowing other objects to directly access its list of dishes): [see the exemplar](https://github.com/makersacademy/takeaway-exemplar/blob/master/lib/menu.rb).